### ISSUE TITLE
Unable to pre-download imagery from Settings anymore 

### ISSUE THREAD

**Issue by mnalis** [Zverik/every_door#710]
It used to work (mostly - see e.g. https://github.com/Zverik/every_door/issues/589) in the past.
 
Now I'm trying in 4.1 and pre-download of satellite/aerial imagery doesn't seem to work at all. EveryDoor 4.1.0 from F-droid.

Loading it on-demand while panning the map while internet is present work just fine, though!

Here is video:

https://github.com/Zverik/every_door/assets/156656/3c74f359-4d88-4132-a506-ff2d7369f9a7

Some important timestamps:
- `0:24` try to click download arrow to download satellite "Esri World Imagery" imagery. Nothing happens (before, IIRC it turned to pie-chart icon showing how much download has progressed).
- `0:35` disable WiFi
- `1:00` see there is **blank  grey screen** instead of satellite background map when trying to switch to it
- `1:18` choose different ("dgu.hr: Croatia 2019-2020") aerial map imagery (after re-enabling WiFi first)
- `1:26`, `01:33`, `01:40` try to download that different aerial. Still no indication that download is happening.
- `1:43` disable WiFi again
- `1:52` see there is **blank  grey screen** instead of aerial background map when trying to switch to it
- `2:19` show **system log** which seems to have some errors
- `2:31` enable WiFi
- `2:41` when trying to switch to aerial background map, it works and loads it without an issue.
- `2:47` disable WiFi again
- `2:50` switch to aerial background map. It shows the aerial map tiles cached at `2:41` normally.

**Comment by Zverik** [comment]
Yes, that's likely. I honestly haven't used the downloading feature since I made it. I will tackle this with #576.

**Comment by pandadeepimpact** [comment]
Mine has the same response too (that pressing the Download button in satellite images does nothing), but I'm surprised that it was even possible in the first place, as none of the editors I have encountered so far has an offline satellite image feature.

That said, I'd like to have that feature returned as well, as satellite imagery can be useful both for navigation and for counterchecking (mismatch between OSM and satellite imagery can quickly reveal outdated features).

**Comment by Zverik** [comment]
Created a separate panel for pre-downloading tiles for 7.0.
